client/src/views/Host.vue

```diff
@@ -189,7 +199,8 @@ export default {
       this.socket.emit('pause-state', pausePackage)
     },
     kickPlayer(clientId) {
-      let playerIndex = this.playerList.findIndex((player) => player.clientId === clientId)
+      let playerIndex = this.playerList
+        .findIndex((player) => player.clientId === clientId)
```

Comment:
Why did we change this?

Reply:
I guess the line was very long so i broke it up. Doesn't change the way the code functions though

Reply:
If you're going to break it here, why not in any other occasions? You should try to follow one set styling if you're going to style it like that.

Reply:
Its usually common in JS to break it there when a line gets too long. If that standard is inconsistently applied I will go through and correct it